projects/packages/videopress/src/client/block-editor/blocks/video/components/videopress-uploader/index.native.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { BlockIcon, MediaPlaceholder } from '@wordpress/block-editor';
+import { useDispatch } from '@wordpress/data';
+import { useCallback, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { isURL, getProtocol } from '@wordpress/url';
+/**
+ * Internal dependencies
+ */
+import { buildVideoPressURL } from '../../../../../lib/url';
+import { VIDEOPRESS_VIDEO_ALLOWED_MEDIA_TYPES } from '../../constants';
+import { description, title } from '../../index';
```

Comment:
@fluiddot, it looks like the `description` import just needs to be removed and then I think all will be good.

Reply:
Ah, my bad 🤦. I've just fixed this in https://github.com/Automattic/jetpack/pull/29461/commits/e720ce221d611535d58441f4d9df8e92e5d32e76. Thanks!